src/cli.ts

```diff
@@ -214,3 +227,36 @@ async function routeGenerator(args) {
     process.exit(1);
   }
 }
+
+async function generateSwaggerAndRoutes(args) {
+  try {
+    const config = await getConfig(args.configuration);
+    if (args.basePath) {
```

Comment:
can you please give args a type so it's not implicitly any? (btw, there will be a PR soon that turns on noImplicitAny in tsconfig)